apps/store/extensions/app/store-common/pages/top_assets.jag

```diff
@@ -75,8 +81,12 @@ require('/modules/store.js').exec(function (ctx) {
     var qtemp = ctx.request.getParameter('q') || '';
     var searchQuery = utilsRequest.formatSearchQuery(qtemp);
     output.searchQuery = decodeURIComponent(searchQuery)||'';
-
-    caramel.render(output);
+    try {
+        metrics.start("org.wso2.store.store-common.top-assets.caramel.render");
+        caramel.render(output);
+    } finally {
+        metrics.stop();
+    }
     //print(page);
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/7ad525f84e84bc725699efe3abcb4d7dab4c32b0)